CONTRIBUTING.md

```diff
@@ -4,48 +4,101 @@ Want to hack on kubernetes? Yay!
 
 ## Developer Guide
 
-We have a [Developer's Guide](docs/devel/README.md) that outlines everything you need to know from setting up your dev environment to how to get faster Pull Request reviews. If you find something undocumented or incorrect along the way, please feel free to send a Pull Request.
+We have a [Developer's Guide](docs/devel/README.md) that outlines everything
+you need to know from setting up your dev environment to how to get faster Pull
+Request reviews. If you find something undocumented or incorrect along the way,
+please feel free to send a Pull Request.
 
 ## Filing issues
 
-If you have a question about Kubernetes or have a problem using it, please read the [troubleshooting guide](docs/troubleshooting.md) before filing an issue.
+If you have a question about Kubernetes or have a problem using it, please
+start with the [troubleshooting guide](docs/troubleshooting.md).  If that
```

Comment:
The actual troubleshooting guide has moved to kubernetes.io. We should delete the one in this repo.